### ISSUE TITLE
Support BLE or work with smart watch \ fitness tracker ?

### ISSUE THREAD

**Issue by AlexGluck** [tiktaalik-dev/blueproximity#7]
I try amazfit bip and xiaomi mi band (1,2,3) in all cases i can't select this devices for lock screen.
May be [that](https://github.com/filug/blueproximity/commit/6081ccb81dc5889ebbdedbaa5551b8f5ec56d4f7#diff-694eef9d69fb3a43a7bc4c7ba3363c4c8935c34bcc5c69e3f95b03866075bda8R1119) can help for add support fit trackers.

P.S. May be i can help with rpm and russian translation (later).

**Comment by tiktaalik-dev** [comment]
Hi @AlexGluck,

Thanks for reaching out :-)

Someone had mentioned similar problems with Bluetooth Low Energy devices but as I don't own any such device I wasn't aware that such issue existed and how to solve it yet. Sorry about that!
Thanks so much for that link to a possible solution! I skimmed through the code and it seems to be relatively simple to implement, provided that library is already compatible with Python 3 (haven't checked it up yet). Would you be willing to test the new code base when I have it, please?
Also, thanks for offering to help with rpm packages! That would be amazing!
About the translation, that would be much appreciated as well. The original language files were not available in the old code but I managed to extract them from the binary files a while ago, so once I push the files to the repository will let you know to ask for your assistance.

Cheers,
Rigo

**Comment by AlexGluck** [comment]
Yep, of course i have "xiaomi mi band 3" and "amazfit bip". Translation will be easy, but for packaging need time.

**Comment by oangelo** [comment]
@tiktaalik-dev, to the best of my knowledge, any smartphone can work as a Bluetooth Low Energy device. Support BLE would be a very nice feature indeed.

**Comment by pbludov** [comment]
This fork has BLE support: https://github.com/filug/blueproximity
